### ISSUE TITLE
Replace temporary rhythm images (4-9) by the real ones

### ISSUE THREAD

**Issue by AnoukZ** [AnoukZ/anoukz.github.io#2]

**Comment by AnoukZ** [comment]
![Ritme01](https://github.com/AnoukZ/anoukz.github.io/assets/77325580/483a9f9e-1b34-4727-b0ac-81d7a1a66b4b)
![Ritme02](https://github.com/AnoukZ/anoukz.github.io/assets/77325580/cd2f7985-44bf-4525-afe7-35a32bee76cd)
![Ritme03](https://github.com/AnoukZ/anoukz.github.io/assets/77325580/3de1dbbe-ec4b-48bf-b050-c8583eead6a7)
![Ritme04](https://github.com/AnoukZ/anoukz.github.io/assets/77325580/2fb3cc96-6433-469a-9f7a-3ca4ad909d06)
![Ritme05](https://github.com/AnoukZ/anoukz.github.io/assets/77325580/67243f88-0882-4895-ba28-c931dfd92481)
![Ritme06](https://github.com/AnoukZ/anoukz.github.io/assets/77325580/12de286d-3645-4c30-b117-ae757151c0d3)
![Ritme07](https://github.com/AnoukZ/anoukz.github.io/assets/77325580/62828550-6169-49df-b811-69c0744dd651)
![Ritme08](https://github.com/AnoukZ/anoukz.github.io/assets/77325580/5998e7ab-967d-4677-a0fb-7b8fcf94d666)
![Ritme09](https://github.com/AnoukZ/anoukz.github.io/assets/77325580/c4e1e442-1db4-4821-8013-dc063b957949)
![Ritme10](https://github.com/AnoukZ/anoukz.github.io/assets/77325580/ad68d6b2-32bf-48a0-8b91-4dd2f10e8715)

**Comment by AnoukZ** [comment]
Received 10 new rhythm images from Anouk, all at 100x146. Replaced the old ones and published the new ones on the website.

Two new issues:
- Not completely satisfied about the resolution, so will create another ticket to use high resolution images (once scaling is done).
- Current max is 9 rhythm images, on a large resolution screen 10 images also fit, but image 10 is not show yet
